src/components/create-schedule-options.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ChangeEvent } from 'react';
+
+import {
+  FormControlLabel,
+  InputLabel,
+  Radio,
+  RadioGroup,
+  Typography
+} from '@mui/material';
+import Stack from '@mui/system/Stack';
+
+import { useTranslator } from '../hooks';
+import { ScheduleInputs } from './schedule-inputs';
+
+export type CreateScheduleOptionsProps = {
+  label: string;
+  name: string;
+  id: string;
+  createType: string;
+  schedule?: string;
+  timezone?: string;
+  handleCreateTypeChange: (
+    event: React.ChangeEvent<HTMLInputElement>,
+    value: string
+  ) => void;
+  handleScheduleChange: (event: ChangeEvent) => void;
+  handleTimezoneChange: (newValue: string | null) => void;
+};
+
+export function CreateScheduleOptions(
+  props: CreateScheduleOptionsProps
+): JSX.Element | null {
+  const trans = useTranslator('jupyterlab');
+
+  const labelId = `${props.id}-label`;
+
+  return (
+    <Stack spacing={4}>
+      <InputLabel id={labelId}>{props.label}</InputLabel>
+      <RadioGroup
+        aria-labelledby={labelId}
+        name={props.name}
+        value={props.createType}
+        onChange={props.handleCreateTypeChange}
+      >
+        <FormControlLabel
+          value="Job"
+          control={<Radio />}
+          label={trans.__('Run now')}
+        />
+        <Typography sx={{ fontSize: '90%', marginLeft: '2.5em' }}>
```

Comment:
It is important for us to pull all typography from the JupyterLab theme (size, line height, color). In an MUI context, we could either use a plain HTML element and style it using the JupyterLab theme variables, or we could put it inside the `Typography` component and edit the theme to have MUI use the JupyterLab font style information. My only concern about that is that MUI may be using the typography styling in other contexts that we don't want to restyle.